apps/test/unit/templates/rubrics/EvidenceLevelsTest.jsx

```diff
@@ -14,62 +12,28 @@ const DEFAULT_PROPS = {
 };
 
 describe('EvidenceLevels', () => {
-  it('renders evidence levels when feedback available', () => {
-    const wrapper = shallow(
-      <EvidenceLevels {...DEFAULT_PROPS} canProvideFeedback />
-    );
-    expect(wrapper.find('Heading6').length).to.equal(1);
-    expect(wrapper.find('Heading6').props().children).to.equal(
-      'Assign a Rubric Score'
-    );
-    expect(wrapper.find('Memo(RadioButton)').length).to.equal(
-      DEFAULT_PROPS.evidenceLevels.length
-    );
-    expect(wrapper.find('BodyThreeText').length).to.equal(
-      DEFAULT_PROPS.evidenceLevels.length
-    );
-    const lastEvidenceLevel =
-      DEFAULT_PROPS.evidenceLevels[DEFAULT_PROPS.evidenceLevels.length - 1];
-    expect(wrapper.find('BodyThreeText').at(0).props().children).to.equal(
-      lastEvidenceLevel.teacherDescription
-    );
-    expect(wrapper.find('Memo(RadioButton)').at(0).prop('label')).to.equal(
-      UNDERSTANDING_LEVEL_STRINGS[lastEvidenceLevel.understanding]
-    );
+  it('renders teachers view of evidence levels when the user can provide feedback', () => {
+    const wrapper = shallow(<EvidenceLevels {...DEFAULT_PROPS} />);
+    expect(wrapper.find('EvidenceLevelsForTeachers').length).to.equal(1);
+    expect(
+      wrapper.find('EvidenceLevelsForTeachers').props().canProvideFeedback
```

Comment:
why is this undefined? The name of this test suggests it should be true?

Reply:
That test should have been renamed to "renders teachers view of evidence levels when the user can **not** provide feedback".  It felt weird to test that something was `undefined` but I checked to makes sure that we do it in other places in our codebase (which we do).